Makefile

```diff
@@ -97,7 +105,14 @@ check-node-version:
 # Install dependencies. If deps are up to date this is almost instant, so we can
 # run before every other target with very little DX cost.
 .PHONY: install # PRIVATE
-install: check-node-version
+install: | check-node-version
```

Comment:
This is cool! How does this syntax work?

Reply:
this make the prerequisites [order-only](https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html). 

we actually only need them for the `%` target, because it maintains the value of `$@` after the prerequisites have run. without it, the value changes to `"makefile"`, which i'm _guessing_ is because the prerequisites have no actual output, though i'm not 100% sure. i know this makes it work though!

i've removed them now from the other targets, as they don't have an effect